src/analysis/specific_limits.lean

```diff
@@ -63,6 +63,24 @@ lemma tendsto_pow_at_top_at_top_of_one_lt [linear_ordered_ring α] [archimedean
   tendsto (λn:ℕ, r ^ n) at_top at_top :=
 sub_add_cancel r 1 ▸ tendsto_add_one_pow_at_top_at_top_of_pos (sub_pos.2 h)
 
+/-- The function `x^n` tends to `+∞` at `+∞` for any positive natural `n`.
+A version for positive real powers exists as `tendsto_rpow_at_top`. -/
+lemma tendsto_pow_at_top {n : ℕ} (h : 1 ≤ n) : tendsto (λ (x:ℝ), (x^n)) at_top at_top :=
```

Comment:
Can you frame this lemma for linearly ordered fields, or some such?  Also, I thought a little and am not sure whether this lemma (and the next) belong in `analysis.specific_limits` or `topology.algebra.ordered`.

Reply:
Maybe, in fact, linearly ordered rings.  Can the current usage of `fpow_le_of_le` can be replaced by `pow_le_pow `, and the usage of `fpow_one` by `pow_one`?  However, the next lemma would still have to be stated for linearly ordered fields, because `fpow_neg` involves taking inverses,